### PR TITLE
add current proper link

### PR DESCRIPTION
**Pull request by Umangvaghela**
add a proper link in plugin main file

### CODE REVIEW

**Comment by Umangvaghela** [comment]
I have found in some another place static link is HTTP://example.com while this site actually secure.

**Comment by jeffstieler** [comment]
Good catch @Umangvaghela and thanks for the pull request!

This has highlighted that we need to update our translation files, so I'm going to switch the base to a branch off `master` and update the other files there before merging.

Thanks again. 🎉

File: woocommerce-services.php [hunk 1]
```diff
@@ -154,6 +154,7 @@ class WC_Connect_Loader {
 		protected $nux;
 
 		/**
+		/**
```
Comment (jeffstieler) [inline woocommerce-services.php]: Can you remove this extra comment line? Thanks!

**Comment by Umangvaghela** [comment]
Hello @jeffstieler 

Sir, I worked on your change request and resolve it. When you get a chance review it.

Thanks

**Comment by Umangvaghela** [comment]
Hello @jeffstieler

Sir my patch is merge but my name is not shown in contribution.

Thanks

**Comment by jeffstieler** [comment]
Hi @Umangvaghela - I switched the base for the merge into a branch that isn't `master` because there are some other changes that need to be made for our `i18n` layer. Once [that branch](https://github.com/Automattic/woocommerce-services/compare/update/translations-and-non-ssl-urls) is merged into `master`, you'll show up in the contributors list.

Thanks again for your pull request!

**Comment by jeffstieler** [comment]
@Umangvaghela - you're in! 😄 

https://github.com/Automattic/woocommerce-services/graphs/contributors

**Pull request merged**